source/common/ssl/context_config_impl.h

```diff
@@ -69,9 +81,10 @@ class ContextConfigImpl : public virtual Ssl::ContextConfig {
   const std::string cipher_suites_;
   const std::string ecdh_curves_;
   Secret::TlsCertificateConfigProviderSharedPtr tls_certficate_provider_;
-  Common::CallbackHandle* secret_update_callback_handle_;
+  Common::CallbackHandle* tls_certificate_update_callback_handle_;
```

Comment:
just add {} next to the raw pointer to initailize it.

Reply:
Done.